tests/Integration/inc/classes/subscriber/Optimization/class-critical-css-subscriber/generateCriticalCssOnActivation.php

```diff
@@ -0,0 +1,55 @@
+<?php
+namespace WP_Rocket\Tests\Integration\inc\classes\subscriber\Optimization\Critical_CSS_Subscriber;
+
+use FilesystemIterator;
+use Brain\Monkey\Filters;
+use Brain\Monkey\Functions;
+use WP_Rocket\Tests\Integration\FilesystemTestCase;
+
+/**
+ * @covers \WP_Rocket\Subscriber\Optimization\Critical_CSS_Subscriber::generate_critical_css_on_activation
+ * @group  Subscribers
+ * @group  CriticalCss
+ */
+class Test_GenerateCriticalCssOnActivation extends FilesystemTestCase {
+
+	protected $structure = [
+		'critical-css' => [
+			'1' => [
+				'.'            => '',
+				'..'           => '',
+				'critical.css' => 'css content',
```

Comment:
The `'css content'` here will confuse us later. Can you add a little css here please?